x/axelarnet/keeper/msg_server.go

```diff
@@ -57,6 +57,14 @@ func (s msgServer) CallContract(c context.Context, req *types.CallContractReques
 		return nil, fmt.Errorf("%s is not a registered chain", req.Chain)
 	}
 
+	if !chain.IsFrom(evmtypes.ModuleName) {
+		return nil, fmt.Errorf("non EVM chains are not supported")
+	}
```

Comment:
Why do we want this? Can't someone send to a cosmos chain via `CallContract` and then `ExecuteMessage`?

Reply:
Not at the moment. `ExecuteMessage` doesn't support axelar as the sender. Until we fix it there, we shouldn't allow setting the message under `Sent` status for cosmos chains